docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -1371,6 +1371,27 @@ If you are compiling a C/C++ compiler, where practical make the generated compil
 | <span id="--enable-host-bind-now">`--enable-host-bind-now`</span>         | GCC 14.0.0       | Build the compiler executables with [`-Wl,-z,now`](#-Wl,-z,now) |
 | <span id="CLANG_DEFAULT_PIE_ON_LINUX">`CLANG_DEFAULT_PIE_ON_LINUX`</span> | Clang 14.0.0 | Turn on [`-fPIE`](#-fPIE_-pie) and [`-pie`](#-fPIE_-pie) by default for binaries produced by the compiler |
 
+## What should you do when compiling linkers?
+
+If you are compiling a linker, where practical make the generated linker's default options the *secure* options. The below table summarizes relevant options that can be specifed when building GNU Binutils that affect the defaults of the linker:
+
+| Linker Flag                   | Supported since  | Description                                                       |
+|:--- |:---:|:---- |
+| <span id="--disable-default-execstack">`--disable-default-execstack`</span> | Binutils 2.42 | Require the `GNU_STACK` ELF note for executable stacks, rather than enabling it by default. |
```

Comment:
The _"it"_ in the description makes the wording a bit awkward, particularly since the _"executable stacks"_ that _"it_" refers to is in plural. 

Also `--disable-default-execstack` | `--enable-default-execstack=no` was already added in [Binutils 2.39](https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;a=blob_plain;f=ld/NEWS;hb=refs/tags/binutils-2_39) .

```suggestion
| <span id="--disable-default-execstack">`--disable-default-execstack`</span> | Binutils 2.39 | Disable creation of executable stacks unless requested with `GNU_STACK` ELF note. |
```

Reply:
I wonder how I concluded this, I distinctly remember checking `git tag --contains`. Thanks!

Reply:
I understand what happened now -- e922d1eaa3774a68c96eae01e0fd08f8a30cda8c (2.42) adds related but different options, and I mixed the versions up in the table:
```
 +* Added --warn-execstack-objects to warn about executable stacks only when an
+  input object file requests one.  Also added --error-execstack and
+  --error-rxw-segments options to convert warnings about executable stacks and
+  segments into errors.
+
+  Also added --enable-error-execstack=[yes|no] and
+  --enable-error-rwx-segments=[yes|no] configure options to set the default for
+  converting warnings into errors.
```